xc7/utils/prjxray_routing_import.py

```diff
@@ -111,7 +111,49 @@ def populate_bufg_rebuf_map(conn):
 
     cur = conn.cursor()
 
-    # Find nodes touching rebuf wires.
+    # Find CLK_HROW_TOP_R, CLK_HROW_TOP_R and REBUF tiles.
```

Comment:
Please move the map construction to a function.  Consider writing a pytest for the function?

Reply:
I moved it to a function. However developing a test is problematic as the function relies on data from the `channels.db`